lib/landrush/action/redirect_dns.rb

```diff
@@ -24,7 +24,7 @@ def redirect_dns
       def _target_host
         case provider
         when :virtualbox then
-          '10.0.2.2'
```

Comment:
Looking at the other cases, it seems we are after the gateway here. The IP to reach the host. That's imo 10.0.2.2. I am tempted to leave it the way it is, since it works. I have not seen any actual errors around this. WDYT?